src/pages/Explore/Explore.jsx

```diff
@@ -0,0 +1,28 @@
+import { useEffect } from "react";
+import { Sidebar } from "../../Components";
+import axios from "axios";
+import { useDispatch, useSelector } from "react-redux";
+import { AddPost } from "../../reducer/postSlice";
+
+export const Explore = () => {
+  const { post } = useSelector((store) => store.posts);
+  console.log(post);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    (async () => {
+      try {

```

Comment:
for api call use createAsyncThunk provided by redux

Reply:
yeah , but I am not using any loader , so It doesn't make any sense